ibis/backends/base/sql/compiler/query_builder.py

```diff
@@ -232,6 +232,9 @@ def _translate(self, expr, named=False, permit_subquery=False):
         )
         return translator.get_result()
 
+    def __hash__(self):
```

Comment:
Why does this need to be hashable?

Reply:
Because the object must be hashable now to be a subclass of comparable.

Reply:
Got it, thanks for explaining.

Reply:
Actually I added defailt implementation for `Comparable.__hash__()` now to defer it to `super().__hash__()`